src/views/Product/queries.ts

```diff
@@ -54,6 +54,7 @@ export const productDetailsQuery = gql`
     product(id: $id) {
       ...BasicProductFields
       description
```

Comment:
If we use `descriptioniJson` I guess we should omit getting `description` to not overfetch data which we do not need :)